### PR TITLE
Test PR so I can enjoy the diff

### PR DESCRIPTION
**Pull request by vn-ki**

### CODE REVIEW

File: sunpy/time/timerange.py [hunk 1]
```diff
@@ -65,11 +67,13 @@ def __init__(self, a, b=None):
             y = b
 
         if isinstance(y, u.Quantity):
-            y = timedelta(seconds=y.to('s').value)
+            y = TimeDelta(y)
+
+        # TODO: Support datetime.timedelta?
```
Comment (vn-ki) [inline sunpy/time/timerange.py]: @cadair @nabobalis Should we support the case where y (ie b) is `datetime.timedelta`?

File: sunpy/time/timerange.py [hunk 2]
```diff
@@ -185,7 +189,8 @@ def _duration(self):
         -------
         value : `astropy.units.Quantity`
         """
-        return self.dt.total_seconds() * u.s
+        # TODO: Remove this?
+        return self.dt
```
Comment (vn-ki) [inline sunpy/time/timerange.py]: Should we remove `_duration` as `dt` can serve the serve purpose?

File: sunpy/time/astropy_time.py [hunk 3]
```diff
@@ -93,3 +94,10 @@ def strftime(self, format_spec):
                 return formatted_strings[0]
             else:
                 return np.array(formatted_strings).reshape(self.shape)
+
+
+def is_time_equal(t1, t2):
```
Comment (Cadair) [inline sunpy/time/astropy_time.py]: is it a good idea to make this part of the public API, is it not just a work around for the comparison bug in astropy?

Comment (vn-ki) [reply sunpy/time/astropy_time.py]: I'll add a `_`. :+1:

Comment (Cadair) [reply sunpy/time/astropy_time.py]: also remove it from the `__all__`

Comment (vn-ki) [reply sunpy/time/astropy_time.py]: Then how will I import it?

EDIT: Got it.

File: sunpy/time/timerange.py [hunk 4]
```diff
@@ -269,7 +262,7 @@ def split(self, n=2):
         previous_time = self.start
         next_time = None
         for _ in range(n):
-            next_time = previous_time + self.dt // n
+            next_time = previous_time + self.dt / 2
```
Comment (vn-ki) [inline sunpy/time/timerange.py]: Not 2, but `n`

**Pull request closed without merging**